src/views/network/CreateL2NetworkForm.vue

```diff
@@ -125,7 +125,7 @@
               :placeholder="this.$t('label.networkofferingid')"
               @change="val => { this.handleNetworkOfferingChange(this.networkOfferings[val]) }">
               <a-select-option v-for="(opt, optIndex) in this.networkOfferings" :key="optIndex">
-                {{ opt.name || opt.description }}
+                {{ opt.displaytext || opt.description }}
```

Comment:
@ravening can this be: `opt.displaytext | opt.name | ...` to cover for the case if/when displaytext is empty?